### ISSUE TITLE
Fix(refector): Minor fixes for long-term-storage-retrieval 

### ISSUE THREAD

**Issue by patrikgrenfeldt** [Clinical-Genomics/cg#1515]
## Description

Address comments on code found in PR https://github.com/Clinical-Genomics/cg/pull/1486. Not addressed in PR since the developer is no longer working for CG.

<img width="764" alt="image" src="https://user-images.githubusercontent.com/6697636/175889136-0366769e-df91-4ca0-bd85-0b2523f47e06.png">

<img width="746" alt="image" src="https://user-images.githubusercontent.com/6697636/175889207-34512c0e-cff8-4cee-bdcc-d7981b36db92.png">

<img width="746" alt="image" src="https://user-images.githubusercontent.com/6697636/175889260-115c5f17-9ade-48e3-9514-279f31cb2239.png">

<img width="746" alt="image" src="https://user-images.githubusercontent.com/6697636/175889322-2f298f86-f15e-4361-bcad-6debdfd8eaaf.png">

<img width="746" alt="image" src="https://user-images.githubusercontent.com/6697636/175889386-b5f78865-37a3-41c1-b7dd-bc7633d6e0ec.png">

## Suggested solution

Fix above comments 

## This can be closed when

- [ ] Comments in description addressed

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by thelander** [comment]
@karlnyr was everything in this issue fixed in PR https://github.com/Clinical-Genomics/cg/pull/1517? Or are there still issues to be considered? 😊

**Comment by karlnyr** [comment]
@thelander I believe that the stort by latest is not fixed in the query :)

**Comment by ChrOertlin** [comment]
@karlnyr this is assigned to you. Can this be closed?

**Comment by ChrOertlin** [comment]
Issue is not addressed yet. Added to refinement 19-06-2024

Solution that needs to be implemented is a sorting of the pdc output as attempted here: https://github.com/Clinical-Genomics/cg/pull/1557

**Comment by Vince-janv** [comment]
Closing in favour of clearer and updated issue: https://github.com/Clinical-Genomics/cg/issues/2396
